components/HeroSection.vue

```diff
@@ -129,8 +196,12 @@ section {
 }
 
 .cta-button {
-  padding: 1.25rem 1.5rem;
-  width: 14rem;
+  padding: 0;
+  width: 14.375rem;
+  height: 3.125rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
```

Comment:
can we use `margin: 0 auto`

Reply:
I think justify and align also works well with flex